requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy>=1.16.0
 pandas>=0.25.0,<1.1.0
 pvlib>=0.8.0
-scipy>=1.3.0
+scipy>=1.5.0
```

Comment:
Maybe think about this for a moment - pvlib is on scipy >= 1.2.0, and it's a guiding principle here that pvanalytics and pvlib play nice together. Do we want to bump the scipy version? Or, program around the deprecation warning?

Reply:
PVLib tests pass at scipy 1.5.2. I'm personally okay with the different requirement, so long as it satisfies the >= cirteria. We already depend on scipy > 1.2 for the `median_absolute_deviation` function.

Reply:
As someone that recently started trying to use pvanalytics alongside other code, I have two comments:

1. this is a big jump to a newish scipy version (4 months), and would make me think twice about installing pvanalytics in some places and prevent it in others. 
2. it would help to know the pvanalytics policy for dependencies. My suggestion is following [NEP-29](https://numpy.org/neps/nep-0029-deprecation_policy.html) for the core pydata packages. I'm not sure what to recommend for a pvlib version policy.

Reply:
Thanks @wholmgren. I was not aware of NEP-29, and I certainly see the benefits for a predictable approach to dependency management. My approach has been ad-hoc thus far, so I would not be opposed to adopting the policy laid out in NEP-29: all minor versions back 2 years.

That will mean a bit of re-working since some versions have crept higher than that would allow. For instance, to support scipy==1.2 we will need to add `statsmodels` as a dependency and use `statsmodels.robust.scale.mad()` here.

Reply:
Ok to add statsmodels. As far as the median absolute deviation statistic, an imprecise value is good enough here.